src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala

```diff
@@ -46,6 +49,7 @@ object ContainerPlugin extends AutoPlugin {
       Seq(libraryDependencies ++= (containerLibs in conf).value.map(_ % conf)) ++
       inConfig(conf)(Seq(
         start              := (startTask dependsOn webappPrepare).value
+      , develop            := (developTask dependsOn webappPrepare).value
```

Comment:
Calling `webappPrepare` is going to be confusing/a source of bugs by packaging dependencies into the webapp only for them to be in the system classpath instead. Infact depending on the webapp classloader delegation model this won't work at all.

A new webappPrepare task that doesn't package dependencies into to `WEB-INF/lib`/`WEB-INF/classes` is necessary as well.